### PR TITLE
Return error code when the user has no valid submission

### LINKED ISSUE #47: verify-turnin should exit with non-zero value when the user has no valid submission
```
$ verify-turnin ex1@zakkak
 You have not turned in ex1 for zakkak!
$ echo $?
0
```

while it should return a non-zero value.

### PR DESCRIPTION
**Pull request by innerout**
Adjusted the error code when there is no valid submission.
Tested locally to verify that the change works.
Closes #47

### CODE REVIEW

**Review by zakkak — APPROVED** [review]
Thank you for the fix @innerout .

Could you please also add a test for it in https://github.com/zakkak/turnin/blob/8cf29460729b45bbd974fa5a30ccd47f4b859b66/.github/workflows/main.yml#L35-L39 ?

Something like `verify-turnin ex-not-exists@$COURSE || test $? -ne 0`

**Comment by innerout** [comment]
Yeah, sure!

**Comment by zakkak** [comment]
Great, thanks. Please also make sure to add `scripts/**` to the list of paths in:

https://github.com/zakkak/turnin/blob/8cf29460729b45bbd974fa5a30ccd47f4b859b66/.github/workflows/main.yml#L5-L13

**Comment by innerout** [comment]
Testing the GitHub workflow. I will leave a comment when the test is ready.

**Comment by innerout** [comment]
I think it is ok now.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -36,4 +39,7 @@ jobs:
         run: |
           verify-turnin ex1@$COURSE
           sudo -u $COURSE find /home/$COURSE/TURNIN/ex1/
-          sudo -u $COURSE find /home/$COURSE/TURNIN/ex1/ | grep `whoami` > /dev/null
\ No newline at end of file
+          sudo -u $COURSE find /home/$COURSE/TURNIN/ex1/ | grep `whoami` > /dev/null
+      - name: Verify turn in without submission
+        run: |
+          verify-turnin ex2@$COURSE || test $? -ne 0
```
Comment (zakkak) [inline .github/workflows/main.yml:45]:
```suggestion
      - name: Verify turn in without submission
        run: |
          verify-turnin ex2@$COURSE || test $? -ne 0
      - name: Verify turn in in non existing directory
        run: |
          verify-turnin ex3@$COURSE || test $? -ne 0
      - name: Verify turn in in non existing course
        run: |
          verify-turnin ex1@mycourse || test $? -ne 0
```

**Pull request merged**